webpack/ForemanRhCloudFills.js

```diff
@@ -18,10 +18,10 @@ const fills = [
     weight: 400,
   },
   {
-    slot: 'details-cards',
+    slot: 'host-overview-cards',
```

Comment:
Which version of Foreman changed the slot name?

Reply:
@ShimShtein The Foreman change https://github.com/theforeman/foreman/pull/9189 is linked to https://projects.theforeman.org/issues/33010 which is "Fixed in Releases" 3.3.0.

Reply:
btw, if you use the old slot name, the card weight won't work properly and the card will always be last. That's why both changes are here.